wire-ios/Wire-iOS/Sources/UserInterface/Settings/ColorPickerView.swift

```diff
@@ -0,0 +1,83 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+import WireSyncEngine
+
+struct ColorPickerView: View {
+
+    @State var selectedColor: AccentColor?
+
+    let colors: [AccentColor]
+    var onColorSelect: ((AccentColor) -> Void)?
+
+    private let colorViewSize: CGFloat = 28
+    private let colorViewCornerRadius: CGFloat = 14
+
+    var body: some View {
+        NavigationView {
```

Comment:
Why do we have a navigation view?

Reply:
So we can provide the navigation title with the correct font size. There isn't another workaround for this atm. If we don't do it this way the font size for the navigation bar title is too small.

Reply:
I suggest then to add a comment explaining this.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/310ae500a563d65eb3ecb9982330f024d6184876